### PR TITLE
PR AT_14.04.06 | <Multi-configuration project > Configure > General> | Verify possibility to disable the project through configure 

### PR DESCRIPTION
**Pull request by JuliaP2023**
https://trello.com/c/LvNxqeBx/2100-at140406-multi-configuration-project-configure-general-verify-possibility-to-disable-the-project-by-clicking-configure-on-the-le

### CODE REVIEW

**Comment by JuliaP2023** [comment]
[AT_14.04.06 | <Multi-configuration project > Configure > General> | Verify possibility to disable the project by clicking configure on the left side panel inside MCProject](https://trello.com/c/LvNxqeBx/2100-at140406-multi-configuration-project-configure-general-verify-possibility-to-disable-the-project-by-clicking-configure-on-the-le)

File: cypress/e2e/tests/multiConfigurationProjectConfigure.cy.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import projectData from "../../fixtures/pom_fixtures/multiConfigurationProjectCo
 
 describe('multiConfigurationProjectConfigure', () => {
   const homePage = new HomePage();
+
```
Comment (SiarheiMSV) [inline cypress/e2e/tests/multiConfigurationProjectConfigure.cy.js:9]: Pls, delete this empty row

File: cypress/pageObjects/MultiConfigurationProjectPage.js [hunk 2]
```diff
@@ -73,5 +74,7 @@ class MultiConfigurationProjectPage {
         this.getAddEditDescriptionBtn().click();
         return this;
     };
+
```
Comment (SiarheiMSV) [inline cypress/pageObjects/MultiConfigurationProjectPage.js]: pls, delete this empty row

**Comment by SiarheiMSV** [comment]
https://trello.com/c/LvNxqeBx/2100-at140406-multi-configuration-proje… #1112 rename PR AT_14.....

File: cypress/pageObjects/MultiConfigurationProjectPage.js (hunk 2 above)
Comment (JuliaP2023) [reply cypress/pageObjects/MultiConfigurationProjectPage.js:80]: Fixed

File: cypress/e2e/tests/multiConfigurationProjectConfigure.cy.js (hunk 1 above)
Comment (JuliaP2023) [reply cypress/e2e/tests/multiConfigurationProjectConfigure.cy.js]: Fixed

**Review by SiarheiMSV — APPROVED** [review]

File: cypress/e2e/tests/multiConfigurationProjectConfigure.cy.js [hunk 3]
```diff
@@ -74,4 +74,17 @@ describe('multiConfigurationProjectConfigure', () => {
...
+      cy.createMultiConfigurationProject(newItemPageData.multiConfigurationProjectName);
+      homePage
+        .clickMultiConfigProjectNameLink(newItemPageData.multiConfigurationProjectName)
+        .clickConfigureSideMenuLink()
+        .clickEnableDisableSwitch()
+        .clickSaveButton()
+        .getWarningText()
+        .should('be.visible')
+        .and('contain', projectData.warningMessage)
+        .and('have.css', 'color', 'rgb(254, 130, 10)')
```
Comment (Rom-T) [inline cypress/e2e/tests/multiConfigurationProjectConfigure.cy.js:88]: Please move `rgb(254, 130, 10)` in the fixtures

Comment (JuliaP2023) [reply cypress/e2e/tests/multiConfigurationProjectConfigure.cy.js]: done

**Review by Rom-T — APPROVED** [review]

**Pull request merged**